pages/authenticate.tsx

```diff
@@ -4,6 +4,7 @@ import { useStytchUser, useStytch } from '@stytch/nextjs';
 
 const OAUTH_TOKEN = 'oauth';
 const MAGIC_LINKS_TOKEN = 'magic_links';
+const RESET_LOGIN = 'login';
```

Comment:
I think we're missing `reset_password` type.

Should be both `login` which equals the "EML escape hatch" in the UI and `reset_password` that punts you into the reset password flow.

Reply:
I have different redirect URLs, so resets just go to a completely different page. Reset logins go to the same page as a normal magic link/OAuth directs to